### ISSUE TITLE
Identify "Tier 3" level information for each standard

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#63749]

**Comment by briandeconinck** [comment]
@erinrwhite In case you don't get to this until I'm already out of office, here's a spreadsheet with some proposed labels for [WCAG](https://docs.google.com/spreadsheets/d/1-GkiPlrF1uUyjivIa0W1xJmHCxYMKXOZavHbOzRUoDg/edit#gid=401865301) and for what I'm calling [interaction modality](https://docs.google.com/spreadsheets/d/1-GkiPlrF1uUyjivIa0W1xJmHCxYMKXOZavHbOzRUoDg/edit#gid=0), which is my arbitrary broadening of devices to include some stuff that's not strictly speaking a device.

**Comment by shiragoodman** [comment]
thank you for sharing this, @briandeconinck !

**Comment by erinrwhite** [comment]
@briandeconinck thank you so much for these! I'm generating reference link for the WCAG tags. Confirming, should I reference [WCAG 2.2](https://www.w3.org/TR/WCAG22/) documentation or an earlier version?

**Comment by erinrwhite** [comment]
@briandeconinck Thank you! Should we keep the items that are new in 2.2 or add those in down the line?

**Comment by shiragoodman** [comment]
I'd recommend waiting. There's a lot we'll need to do when 2.2 is enforced across the VA. No need to make the effort now.

**Comment by erinrwhite** [comment]
@it-harrison Can you take a look at the [suggested descriptive tags](https://docs.google.com/spreadsheets/d/1q6UGLEd8re94B5o69EFDz2HNMHw6WvdAHl46FHYsA9s/edit#gid=500057009) I have so far?

**Comment by erinrwhite** [comment]
@briandeconinck @allison0034 Sharing [suggested descriptive tags](https://docs.google.com/spreadsheets/d/1q6UGLEd8re94B5o69EFDz2HNMHw6WvdAHl46FHYsA9s/edit#gid=500057009) with you before Brian heads out on leave - thank you again Brian for the list of WCAG and modality tags.

Big questions: 
- Are there any broad categories of information missing? Are we in the ballpark?
- Do you have suggested reference links for the modality and/or context tags?
- Anything glaringly unusual or confusing that should be addressed?

cc @shiragoodman @it-harrison 

thanks team!

**Comment by shiragoodman** [comment]
@erinrwhite this looks AWESOME. Thank you so much for putting in the time and effort. I think this will be well received by OCTO practice area leads.

- Are there any broad categories of information missing? Are we in the ballpark?
   - I think this is a great first draft. If there are any "types" missing, we can always add on later. 
- Do you have suggested reference links for the modality and/or context tags?
   - (thinking from the perspective of how these "reference links" will be communicated back to the VFS team in their Staging Review issue ticket) if we don't have suggested reference links, and if the design system doesn't already cover and/or define what the modality and/or context are, then maybe in place of links we could just use the "notes" as definitions. We're relying on the links to communicate the definition and/or more information, but if we don't have them, then maybe we just fill in the blanks ourselves?
- Anything glaringly unusual or confusing that should be addressed?
   - the only thing that comes to mind is that there may already be labels in github for some of the modality tags that are being used by other a11y specialists. If so, we may want to reach out to the a11y community of practice and let them know of our new tags so they're aware. They can choose to align to our tags if they'd like to. We can consult with Martha.

**Comment by allison0034** [comment]
@shiragoodman @erinrwhite these look good. I assume as we do reviews we will discover we need to add more tertiary labels. That wont be a problem correct? I specifically think there will be some around design principals.

**Comment by shiragoodman** [comment]
@allison0034 correct. We can always add more later, but I think this is good enough for now :)
